scripts/install_gradio.sh

```diff
@@ -6,4 +6,7 @@ source scripts/helpers.sh
 pip_required
 
 echo "Installing Gradio..."
-pip install -e .
\ No newline at end of file
+pip install -e .
+
+echo "Installing Gradio Client..."
+pip install -e client/python
```

Comment:
We use `scripts/install_gradio.sh` in a GH action where we don't want to install client/python from scratch. But that's okay, I'll just modify the GH action and push to this branch

Reply:
sounds good, thanks!